src/index.js

```diff
@@ -0,0 +1,50 @@
+import angular from 'angular';
+import '@ovh-ux/ng-translate-async-loader';
+import 'angular-translate';
+import 'ovh-api-services';
+import 'ui-select';
+import 'angular-ui-bootstrap';
+
+import editComponent from './edit/component';
+import component from './component';
+import choiceComponent from './choice/component';
+import factory from './factory';
+import provider from './provider';
+
+import choiceMatchTemplate from './choice/templates/match.tpl.html';
+import choiceSelectTemplate from './choice/templates/select.tpl.html';
+
+import {
+  EDITION,
+  PROTOTYPE_PATH,
+  EMAIL_REGEX,
+} from './constants';
+
+import './index.less';
+
+const moduleName = 'ngOvhContact';
+
+angular
+  .module(moduleName, [
+    'internationalPhoneNumber',
```

Comment:
You defined `international-phone-number` as a peer dependency but shouldn't be also imported?

Reply:
Should be imported with `script-loader` in parent app.